### PR TITLE
[chore][exporter/loki] Replace deprecated format param with format hint.

### PR DESCRIPTION
**Pull request by kovrus**
**Description:** <Describe what has changed.>

Replace the deprecated format configuration parameter (we planned to remove it after v0.59.0) with the format hint.
It is a follow-up on https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/15286

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15351

**Testing:** <Describe what testing was performed and which tests were added.>
Added by https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15351 

**Documentation:** <Describe the documentation added.>
Readme

### CODE REVIEW

File: .chloggen/loki-exporter-logfmt-support.yaml [hunk 1]
```diff
@@ -2,15 +2,15 @@
 change_type: enhancement
 
 # The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
-component: pkg/translator/loki
+component: exporter/lokiexporter
```
Comment (kovrus) [inline .chloggen/loki-exporter-logfmt-support.yaml:5]: Shall we keep a common change log entry for this enhancement?

**Comment by kovrus** [comment]
cc: @domasx2 @mar4uk @gouthamve

**Review by mar4uk — APPROVED** [review]

**Comment by jpkrohling** [comment]
Let's wait to have an alignment with the Loki team. They previously said that JSON would be preferrable, but this might have changed.

**Comment by fatsheep9146** [comment]
any progress?

**Review by domasx2 — APPROVED** [review]

**Comment by kovrus** [comment]
Closing this PR. It is out of sync and the decision on how to proceed with this issue is not yet made.

**Pull request closed without merging**